skins/wmde19/src/components/pages/Error.vue

```diff
@@ -14,8 +13,25 @@
+			<br><br>
+			Imperdiet massa tincidunt nunc pulvinar sapien. Enim lobortis scelerisque fermentum dui faucibus. Pharetra massa massa ultricies mi. Dignissim enim sit amet venenatis urna cursus eget. Porttitor rhoncus dolor purus non enim praesent. Vitae purus faucibus ornare suspendisse sed nisi. Lorem mollis aliquam ut porttitor leo a diam. In tellus integer feugiat scelerisque varius morbi. Leo in vitae turpis massa sed elementum tempus. Auctor urna nunc id cursus metus. Tristique et egestas quis ipsum suspendisse ultrices gravida dictum. Vel pharetra vel turpis nunc eget. Id cursus metus aliquam eleifend mi in nulla posuere sollicitudin. Sed faucibus turpis in eu mi. Urna nunc id cursus metus aliquam eleifend. Felis bibendum ut tristique et egestas quis ipsum.
+			<br><br>
+			Leo a diam sollicitudin tempor. Faucibus purus in massa tempor nec feugiat nisl pretium fusce. Massa placerat duis ultricies lacus sed turpis tincidunt id. Convallis convallis tellus id interdum velit. Sagittis orci a scelerisque purus semper. Quis viverra nibh cras pulvinar mattis nunc sed blandit libero. Gravida quis blandit turpis cursus in hac. Ipsum faucibus vitae aliquet nec. A diam maecenas sed enim ut sem viverra aliquet. Sit amet est placerat in egestas erat imperdiet sed. Id aliquet lectus proin nibh nisl condimentum id. Et netus et malesuada fames ac turpis egestas. Leo a diam sollicitudin tempor id eu nisl nunc mi.
+			<br><br>
+			Lacus suspendisse faucibus interdum posuere lorem. Condimentum mattis pellentesque id nibh tortor. In aliquam sem fringilla ut morbi tincidunt. Velit ut tortor pretium viverra suspendisse. Sapien faucibus et molestie ac feugiat sed lectus. Tempus urna et pharetra pharetra massa. Id consectetur purus ut faucibus pulvinar. Est ultricies integer quis auctor elit sed. Potenti nullam ac tortor vitae purus faucibus ornare. Mi sit amet mauris commodo quis imperdiet massa. In pellentesque massa placerat duis ultricies lacus sed turpis.
+			<br><br>
+			In pellentesque massa placerat duis. Pellentesque eu tincidunt tortor aliquam nulla facilisi. Vulputate eu scelerisque felis imperdiet proin fermentum leo vel orci. Magna ac placerat vestibulum lectus mauris. Ipsum nunc aliquet bibendum enim facilisis gravida neque convallis a. Aenean euismod elementum nisi quis eleifend quam adipiscing vitae proin. Sagittis nisl rhoncus mattis rhoncus urna neque. Arcu odio ut sem nulla pharetra diam sit amet. Sit amet mattis vulputate enim nulla aliquet. A condimentum vitae sapien pellentesque habitant morbi tristique. In tellus integer feugiat scelerisque varius morbi enim nunc faucibus. Faucibus pulvinar elementum integer enim neque volutpat ac tincidunt vitae. Scelerisque varius morbi enim nunc.
+		</p>
+	</div>
```

Comment:
What is this for? Please remove before merging this branch.

Reply:
Of course, just wanted to have content that would stretch to the bottom of the page. Though there is a bit of a problem on pages with very little content, the footer will not be aligned to the bottom of the screen right now (but it also does not do that on production right now).